backend/status/status_source.go

```diff
@@ -0,0 +1,222 @@
+package status
```

Comment:
Should we call this `statussource` or `errorsource`. Thinking about it, status is quite vague and doesn't describe what is this related to.

Reply:
It's not supposed to be used directly by plugin authors at the moment hence why error source is left intact in the backend package. Grafana would rather use status package I think. We might want to deprecate error source in the backend package eventually since it's mainly used for our internal developed plugins. I moved the status package under experimental for now.

Think there are a few other things that could live in status package as well, such as Status, RequestStatus etc but maybe in the future...